tests/__main__.py

```diff
@@ -7,7 +7,7 @@
 """
 Run unit tests. This is invoked by:
 
-$ python -m psutil.tests
+$ python -m tests
```

Comment:
This is outdated. Let's just remove "this is invoked by...."

Reply:
Done!